bin/auth-api/test/auth-routes.test.ts

```diff
@@ -83,7 +85,9 @@ t.test('Auth routes', async () => {
 
     t.test('verify sdf auth token succeeds for whoami', async () => {
       const authData = await decodeAuthToken(validToken);
-      const sdfToken = createSdfAuthToken(authData.userId, 'foo');
+      const user = await getUserById(authData.userId);
+      const workspace = await createWorkspace(user!);
```

Comment:
prob need a linting ignore here for the `!` or you can first check its not empty and throw an error

Reply:
Fixed